### ISSUE TITLE
[Report] Problem with controllers by Bloopair (BUG)

### ISSUE THREAD

**Issue by RS55** [V10lator/NUSspli#368]
I have been using Bloopair from [https://github.com/GaryOderNichts/Bloopair ](https://github.com/GaryOderNichts/Bloopair?form=MG0AV3)since I don't have a gamepad (cheap Wii U without a gamepad) nor Wiimotes (I lent those out), but when I try to install an app using Nusspli, just when the app starts and the first menu appears, depending on which direction I press, it keeps scrolling up or down infinitely and never stops. This only happens with controllers connected through Bloopair. I have tried with a Switch Pro Controller, a third-party (Chinese) controller which has a Switch Pro mode, and a DualShock 3, and none of them work, in my case only with Wiimotes.

**Comment by V10lator** [comment]
Is this on Aroma? Everything (Aroma, Bloopair, NUSspli) latest version?

**Comment by RS55** [comment]
> Is this on Aroma? Everything (Aroma, Bloopair, NUSspli) latest version?

yes, aroma last version, last version of bloopair and last version of NUSspli. Just now I use my Wii Mote but still the same problem. I removed the bloopair to see if that could be causing it, but it's still the same.

**Comment by V10lator** [comment]
First you said controllers connected through bloopair have the issue but Wiimotes work, now you say Wiimotes are broken even after removing boopair. Don't want to sound mean, just want to get the details straight to be able to either recreate this to be able to add debugging code and do a lot to pin point and fix the issue or maybe even be able to easily spot it in code.

Thing is that I'm normally testing with multiple controllers including a original Wiimote and a (N64 NSO) Switch Pro controller whenever something gets changed at input handling but maybe I forgot last time or something... So back to that Wiimote confusion: Just a shoot into the blue but are maybe original Wiimotes broken but original ones work?

**Comment by V10lator** [comment]
Closing cause of no more reply and input handling had been rewritten a bit anyway.
